modules/builds-source-secrets-entitlements.adoc

```diff
@@ -10,20 +10,32 @@ Builds that use Red Hat subscriptions to install content must include the entitl
 
 .Prerequisites
 
-You must have access to Red Hat entitlements through your subscription. The entitlement secret is automatically created by the Insights Operator.
+You must have access to {op-system-base-full} package repositories through your subscription.
 
+The entitlement secret to access these repositories is automatically created by the Insights Operator when your cluster is subscribed.
 
-[TIP]
+[IMPORTANT]
 ====
-When you perform an Entitlement Build using {op-system-base-full} 7, you must have the following instructions in your Dockerfile before you run any `yum` commands:
+You must be a cluster administrator or have permission to access secrets in the `openshift-config-managed` project.
+====
+
+.Procedure
 
+. Copy the entitlement secret from the `openshift-config-managed` namespace to the build's namespace  by using the following YAML content:
++
 [source,terminal]
 ----
-RUN rm /etc/rhsm-host
+$ oc apply -f - <<EOF
+kind: SharedSecret
+apiVersion: sharedresource.openshift.io/v1alpha1
+metadata:
+  name: etc-pki-entitlement
+spec:
+  secretRef:
+    name: etc-pki-entitlement
+    namespace: openshift-config-managed
+EOF
```

Comment:
Appears this was missed in the copy/rebase.

This needs to be the following terminal commands:

```suggestion
$ cat << EOF > secret-template.txt
kind: Secret
apiVersion: v1
metadata:
  name: etc-pki-entitlement
type: Opaque
data: {{ range \$key, \$value := .data }}
  {{ \$key }}: {{ \$value }} {{ end }}
EOF
$ oc get secret etc-pki-entitlement -n openshift-config-managed -o=go-template-file --template=secret-template.txt | oc apply -f -
```